Documentation/FluidTemplates/Index.rst

```diff
@@ -413,7 +413,7 @@ all pages, it can be part of the global website layout. Therefore, file
 
 Move the `<nav> ... </nav>` part from file
 :file:`Resources/Private/Templates/Page/Default.html` to
-`Resources/Private/Layouts/Page/Default.html` as shown here::
+:file:`Resources/Private/Layouts/Page/Default.html` as shown here::
```

Comment:
Can you add `:file:` to all files/paths in this document?

Reply:
done